src/entity.ts

```diff
@@ -501,7 +501,7 @@ export namespace entity {
     };
 
     if (excludeFromIndexes && excludeFromIndexes.length > 0) {
-      excludeFromIndexes.forEach(excludePath => {
+      excludeFromIndexes.forEach((excludePath: string) => {
```

Comment:
Why do we need the type here?  What's the type of `excludeFromIndexes`?

Reply:
Array of strings on an Entity object which, at the moment, is set to type Entity. Interfaces in other parts of the codebase have `excludeFromIndexes` set to boolean. I'd like to make that more precise